settings.gradle

```diff
@@ -9,6 +9,7 @@ include ':languages:brazilian:pack', ':languages:brazilian:apk'
 include ":languages:catalan:pack", ":languages:catalan:apk"
 include ":languages:czech:pack", ":languages:czech:apk"
 include ':languages:english:pack', ':languages:english:apk'
+include ':languages:kachin:pack', ':languages:kachin:apk'
```

Comment:
Sorry I missed this, but you only need this line once, if you would be able to change that.